ph5/clients/tests/test_ph5tostationxml.py

```diff
@@ -310,16 +384,13 @@ def tearDown(self):
         self.mng.ph5.close()
         super(TestPH5toStationXMLParser_resp_load_not_run, self).tearDown()
 
-    def test_create_obs_network(self):
-        self.parser.manager.ph5.read_experiment_t()
-        self.parser.experiment_t = self.parser.manager.ph5.Experiment_t['rows']
-        self.parser.add_ph5_stationids()
-        with LogCapture() as log:
-            log.setLevel(logging.ERROR)
-            self.parser.create_obs_network()
-            self.assertEqual(log.records[0].msg,
-                             'All response file names are blank in response '
-                             'table. Check if resp_load has been run.')
+    def test_read_networks(self):
+        with self.assertRaises(ph5tostationxml.PH5toStationXMLError) as contxt:
+            self.parser.read_networks()
+            self.assertEqual(
+                contxt.exception.message,
+                'All response file names are blank in response '
+                'table. Check if resp_load has been run.')
```

Comment:
I need your opinion on this. When all response filenames are blank in response table it may mean that resp_load hasn't been run, it may be for another cases. Let me know if you want to raise error here or just log it as error or warning.

Reply:
Is it possible to have a PH5 file that contains a response if respload has not been run?

Reply:
resp_load is one step to create PH5. There is still the case with PH5 file with no resp_load run. However that file may not be submitted to DMC because it must be validated before that.

Reply:
If there is no response information in a ph5 experiment than an error exception should be thrown and ph5tostationxml should not output a stationxml file. Although this logic will force data creators to have fairly complete ph5 metadata before the stationxml version of the metadata can be created, this will help prevent incomplete expirments from slipping through the validation process and will flag any incomplete experiments that are currently stored in the IRIS data holdings. Deciding to use an error or warning for this case seems a trade off between the ease of data creation and data extraction. I would argue that an error should be used and the burden should be put onto the data creator, because they have the tools and knowledge to fix the data rather than a random user that may stumble upon this information in the future.

Reply:
If you look at the test above, you can see that it is to test if the function raise error for that case (PH5toStationXMLErrorV) and the specific error message for that. There will be no stationxml file created at all.

Reply:
I think that this exception looks good. We could consider changing the exception's wording to something similar to:

> Response table does not contain any response file names. Check if resp_load has been run.

This is a lesser nuance that is not as important as the logic.